src/debugging/netcore/NetCoreDebugHelper.ts

```diff
@@ -187,8 +188,8 @@ export class NetCoreDebugHelper implements DebugHelper {
             throw new Error(l10n.t('Unable to determine assembly output path.'));
         }
 
-        if (netSdkDebugHelper.isDotNetSdkBuild(debugConfiguration.preLaunchTask) && projectInfo.length >= 5) { // if .NET has support for SDK Build
-            if (projectInfo[4] === 'true') { // fifth is whether .NET supports SDK Containers
+        if (netSdkDebugHelper.isDotNetSdkBuild(debugConfiguration) && projectInfo.length >= 5) { // if .NET has support for SDK Build

```

Comment:
It would probably be cleaner to just override inferAppOutput in NetSdkDebugHelper to implement the SDK specific behavior and rely on polymorphism to apply the correct logic, rather than doing a conditional check here in the base class.